l1-contracts/contracts/bridge/L2WrappedBaseToken.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity 0.8.24;
 import {ERC20PermitUpgradeable} from "@openzeppelin/contracts-upgradeable-v4/token/ERC20/extensions/draft-ERC20PermitUpgradeable.sol";
 
 import {IL2WrappedBaseToken} from "./interfaces/IL2WrappedBaseToken.sol";
-import {IBridgedStandardToken} from "./interfaces/IBridgedStandardToken.sol";
```

Comment:
I rolled back previous changes as we decided not to upgrade L2WETH, but I will have to return those (I likely will upgrade it)